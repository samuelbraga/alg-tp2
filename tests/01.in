10 18
25 40 65 50 55 70 40 40 30 95
1 0 410000
2 1 350000
3 2 250000
4 3 100000
5 2 250000
5 3 250000
5 4 250000
6 5 50000
7 0 420000
7 6 125000
8 0 220000
8 7 220000
9 0 250000
9 1 350000
9 2 90000
9 5 125000
9 6 320000
9 7 90000