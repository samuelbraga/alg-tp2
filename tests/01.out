1495000 1060
1 1 1 1 2 3 1 2 2 4
4 3 100000
5 4 250000
6 5 50000
8 0 220000
8 7 220000
9 1 350000
9 2 90000
9 5 125000
9 7 90000